src/ReactiveUI.Tests/WhenAny/ReactiveNotifyPropertyChangedMixinTest.cs

```diff
@@ -24,6 +24,8 @@ namespace ReactiveUI.Tests
 {
     public class ReactiveNotifyPropertyChangedMixinTest
     {
+        public string? Dummy { get; set; }
```

Comment:
Used for one of the tests, not sure how to make it better.

Reply:
Probably make it internal and use https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=netcore-3.1

Usually put it in assemblyinfo.cs file